README.md

```diff
@@ -1 +1,202 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- [Ruby on Rails](https://rubyonrails.org) - A powerful web application framework for the backend API.
+- [React](https://reactjs.org) - A JavaScript library for building user interfaces on the frontend UI.
+- [PostgreSQL](https://www.postgresql.org) - An open-source relational database.
+- [Bundler](https://bundler.io) - A tool for managing Ruby gem dependencies.
+
+<!-- FEATURES -->
+
+### Key Features <a name="key-features"></a>
+
+- 🔐 **Splash Screen**: Begin your resort journey with confidence! The splash screen features secure authentication. Start your exploring with 
+                         a captivating splash screen that invites you to log in or sign up.
+- 🌏 **Explore Resorts**: Discover a curated selection of resorts, each offering unique experiences and amenities.
+- ✔️ **Reserve Your Spot**: Reserve your desired fav resort.
+- 📅 **Manage Reservations**: Effortlessly manage your resort bookings, including viewing details.
+- 🖥️ **User-Friendly Interface**: Enjoy an user-friendly interface.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+- Soon.. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FRONTEND -->
+
+## 🔗 Link for the Frontend UI <a name="frontend"></a>
+
+- [Resort Vista Frontend](https://github.com/meryemsanem/resort-front-end)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- KANBAN -->
+
+## 🔗 Kanban Board <a name="kanban"></a>
+
+- [Kanban Board](https://github.com/meryemsanem/resort-back-end/projects/1)
+- [Initial Kanban Board](https://user-images.githubusercontent.com/82718562/279994279-135a04c3-6cc8-4814-a12b-964653967c82.png)
+- There are 4 member in our group.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need to have the following installed:
+
+- [Ruby on Rails](https://rubyonrails.org) - A powerful web application framework for the backend API.
+- [Ruby](https://www.ruby-lang.org/en/) - The programming language used by Rails.
+- [PostgreSQL](https://www.postgresql.org) - An open-source relational database.
+- [Bundler](https://bundler.io) - A tool for managing Ruby gem dependencies.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+git clone https://github.com/meryemsanem/resort-back-end.git
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd resort-back-end
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+```bash
+  rails db:create
+  rails db:migrate
```

Comment:
- [ ] When I try to migrate the database according to instructions, this happens:

 
![image](https://github.com/meryemsanem/resort-back-end/assets/57421823/325f47cc-87bd-4870-97e4-cc0a54b8b7d7)

Migration 20231106134949 is failing and this is the reason:

![image](https://github.com/meryemsanem/resort-back-end/assets/57421823/4e986ed4-c6c8-4e02-a845-d8d8e301a57b)

You are adding references the destinations table twice.


This may work for you because your database is already setup.
In order to correctly debug this, drop your local database with `rails db:drop`, then create it and migrate it again to be sure it works without errors.


- [ ] When I correct the last error and go on migrating the database again, this new error happens:


![image](https://github.com/meryemsanem/resort-back-end/assets/57421823/2599ac41-7678-41e7-977e-c52430abe9c1)

This is the reason:

![image](https://github.com/meryemsanem/resort-back-end/assets/57421823/cf0ddcd8-f756-46fd-ba5a-c44a9e73439e)

By the time you want to create references to the destination table, that said table does not exist yet because the migration that creates the destination table is the last migration, and runs after this one.

Kindly fix this error so your users can migrate the database.  Kindly test the migration process with your own local, newly created database (trying to debug this without deleting your own database and creating a new one will hardly work).

_Note: This means I cannot continue reviewing the app's functionality.  Also, I won't be able to check the frontend project's functionality at the time of this review. This will be taken a look at the next review._